CONTRIBUTING.rst

```diff
@@ -304,6 +305,22 @@ Debugging Tips
      $ npm install
      $ node_modules/.bin/grunt build # -> dest/*.global.js
 
+
+Branch Model
+------------
+
+Sphinx project uses following branches for developing.
+
+``master``
+    Used for main development.  All improvement and refactoring, bugfixes are allowed.
+
+``X.Y`` (cf. 1.7)
```

Comment:
Let's try:

    ``X.Y``
      Where ``X.Y`` is the ``MAJOR.MINOR`` release. Used to maintain...